tools/src/bin/benchmark-clvm-cost.rs

```diff
@@ -264,7 +264,69 @@ fn base_call_time_no_nest(a: &mut Allocator, op: &Operator, per_arg_time: f64) -
         num_samples += 1;
     }
 
-    ((total_time - per_arg_time * num_samples as f64) / num_samples as f64).max(100.0)
+    (total_time - per_arg_time * num_samples as f64) / num_samples as f64
+}
+
+// this adds 32 bytes at a time compared to per_byte which adds 5 at a time
+// at the moment this func is only for sha256tree and will need adapting in the future for other uses
+fn time_per_byte_for_atom(a: &mut Allocator, output: &mut dyn Write) -> (f64, f64) {
+    let mut samples = Vec::<(f64, f64)>::new();
+    let dialect = ChiaDialect::new(ENABLE_SHA256_TREE); // enable shatree
+
+    let op_code = a.new_number(63.into()).unwrap();
+    let quote = a.new_number(1.into()).unwrap();
+    let mut atom_str = "ff".repeat(10_000);
```

Comment:
I think it would be simpler to just create the bytes directly, instead of creating a hex string to then convert to bytes.

```suggestion
    let mut atom = [0xff].repeat(10_000);
```